ocean-client/addons/command-center/src/commands/command.ts

```diff
@@ -51,26 +53,47 @@ export abstract class Command {
   }
 
   protected parseBot(): void {
-    if (!this.isUndecided() || this.commandData.length < 2) return
     console.log('parseBot', this.commandData)
+    let botViaCommandFound = false
     switch (this.commandData[1]) {
       case 'maxi':
       case 'vault-maxi':
         this.bot = Bot.MAXI
+        botViaCommandFound = true
         break
       case 'lm':
       case 'reinvest':
       case 'lm-reinvest':
         this.bot = Bot.REINVEST
+        botViaCommandFound = true
         break
     }
 
-    if (this.bot) {
+    if (botViaCommandFound) {
       this.commandData.splice(1, 1)
       console.log('parseBot', this.commandData)
     }
   }
 
+  protected doGeneralBotAvailabilityChecks(): void {
+    // check which bots are available, if only one, we try to execute for that one
+    const bots = this.availableBots.getBots()
+    if (bots.length === 1) {
+      console.log('doGeneralBotAvailabilityChecks', 'available bots = ', bots[0])
+      this.bot = bots[0]
```

Comment:
means if I have only vault-maxi installed, but send `/setPair BTC-DFI reinvest` it will set it for the maxi, right? maybe only do that if bot is undecided?